src/globals/scss/styles.scss

```diff
@@ -6,6 +6,7 @@ $css--font-face: false !default;
 $css--helpers: true !default;
 $css--body: true !default;
 $css--reset: false !default;
+$css--use-layer: true !default;
```

Comment:
Just settled this offline: `v8` we should change the names from `$css--` to `$global--`.
For now, we keep the `$css--use-layer` name for consistency. All the FEDs agreed that the selector nuance is not the focus for distinction. What's more important is the output styles showing up or not.